cli/src/commands/split.rs

```diff
@@ -126,6 +126,10 @@ pub(crate) struct SplitArgs {
     /// source commit description is kept unchanged.
     #[arg(long = "message", short, value_name = "MESSAGE")]
     message_paragraphs: Vec<String>,
+    /// Open an editor to edit the commit's description, even if one was
+    /// provided via --message.
+    #[arg(long, short = 'E')]
```

Comment:
Do we need the short flag? (I didn't notice it when I reviewed `squash --editor`, but `jj describe --edit` doesn't have a short flag.)